### PR TITLE
[sl] Add Pimcore, TYPO3 and CSBill projects

### PR DESCRIPTION
**Pull request by peterkokot**

### CODE REVIEW

File: projects/sl/pimcore.html.twig [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{% block description %}
+    <p>
+
```
Comment (OskarStark) [inline projects/sl/pimcore.html.twig]: why the blank lines here?

Comment (peterkokot) [inline projects/sl/pimcore.html.twig]: English version has them:
https://github.com/symfony/symfony-marketing/blob/master/projects/en/pimcore.html.twig

If I recall correctly that the appropriate format for projects ce202619307385b68972d696cf094081b23676e2

Comment (javiereguiluz) [inline projects/sl/pimcore.html.twig]: @peterkokot that's correct!

Comment (OskarStark) [inline projects/sl/pimcore.html.twig]: fine, but the firs file doesn't have blank lines

Comment (peterkokot) [inline projects/sl/pimcore.html.twig]: Hm, yes, that is also true... Probably also the English templates should be synced with this style. For CSBill project new lines are now added as well. For other files I'll open a separate PR. Thanks for mentioning.

Comment (OskarStark) [inline projects/sl/pimcore.html.twig]: > Thanks for mentioning.

you're welcome 🍺

**Comment by javiereguiluz** [comment]
Thanks @peterkokot.

**Pull request closed without merging**